packages/semantic-tokens/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@fluentui/semantic-tokens",
+  "version": "0.0.0-alpha.1",
+  "description": "Fluent UI Theme semantic-tokens",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "./dist/index.d.ts",
+  "sideEffects": false,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "*",
+    "@fluentui/scripts-api-extractor": "*",
+    "@fluentui/tokens": ">=1.0.0-alpha",
```

Comment:
We use tokens as a devDep as they are imported via the script for backwards compatible overrides.

We can optimize this further by _only_ exporting the variables listed in fluentOverrides.ts file, however for now we simply export all tokens to a flat export in a singular legacy file for tree shaking.

Reply:
We should definitely write up some kind of plan, and think through any cons to the approach of flat re-exporting those tokens. If it's something we should do in the tokens package itself or just scope the flat export via script to this package only. etc.

Reply:
Since we aim to supersede the existing token set, I think keeping it within our semantic-tokens package makes sense, I've now defaulted it to only export the tokens that are actively used as fallbacks so we can reduce bundle size prior to tree shaking also.

If we choose, we can now also remove the devdep on fluentui/tokens package, however we do use it as a safety mechanism to ensure that the tokens being used as fallbacks exist.